### PR TITLE
DASH-353, DASH-354 fix to the missing assignment on dashboard

### PR DESCRIPTION
**Pull request by pushyamig**

### CODE REVIEW

File: dashboard/impl/src/java/org/sakaiproject/dash/logic/DashboardLogicImpl.java [hunk 1]
```diff
@@ -1474,4 +1487,8 @@ public void updateTaskLock(String task) {
 	public void removeTaskLocks(String task) {
 		this.dao.deleteTaskLocks(task);
 	}
+
+	
+
+
```
Comment (ern) [inline dashboard/impl/src/java/org/sakaiproject/dash/logic/DashboardLogicImpl.java]: spaces?

Comment (pushyamig) [inline dashboard/impl/src/java/org/sakaiproject/dash/logic/DashboardLogicImpl.java]: Removed

File: dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java [hunk 2]
```diff
@@ -128,6 +128,22 @@ public boolean addAvailabilityCheck(AvailabilityCheck availabilityCheck) {
...
+		if(log.isDebugEnabled()) {
+			log.debug("updateAvailabilityCheck( " + availabilityCheck.toString() + ")");
+		}
+		try {
+			getJdbcTemplate().update(getStatement("update.AvailabilityCheck"),
+				new Object[]{availabilityCheck.getScheduledTime(),availabilityCheck.getEntityReference()}
+			);
+			return true;
+		} catch (DataAccessException ex) {
+           log.warn("updateAvailabilityCheck: Error executing query: " + ex.getClass() + ":" + ex.getMessage());
```
Comment (zqian) [inline dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java]: indentation here.

File: dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java [hunk 3]
```diff
@@ -846,6 +862,26 @@ public boolean deleteTaskLocks(String task) {
            return new ArrayList<AvailabilityCheck>();
 		}
 	}
+	public boolean isScheduleAvailabilityCheckMade(AvailabilityCheck availabilityCheck){
```
Comment (zqian) [inline dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java]: select.AvailabilityChecks.entry only takes one param: entityRef. I think only this param is needed here for isScheduleAvailabilityCheckMade()

**Comment by dlhaines** [comment]
Looks good.

File: dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java (hunk 2 above)
Comment (pushyamig) [inline dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java]: Done

File: dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java (hunk 3 above)
Comment (pushyamig) [inline dashboard/impl/src/java/org/sakaiproject/dash/dao/impl/DashboardDaoImpl.java]: Done

**Comment by juanjmerono** [comment]
Why are those jiras in status verified and resolution fixed?
https://jira.sakaiproject.org/browse/DASH-353
https://jira.sakaiproject.org/browse/DASH-354

They should be in status Open and resolution Not Resolved, and will be solved when this PR is merged in master. Also they must have "11 status" field with value Merge to be included in 11.x branch.

**Comment by pushyamig** [comment]
We back ported changes to our application, our tester did not realize it and then closed. I have reopened both

**Comment by juanjmerono** [comment]
Could you squash this into a single commit please?

**Comment by pushyamig** [comment]
@juanjmerono , i have created a new pull request https://github.com/sakaiproject/sakai/pull/1800 for the changes i made here. The had some problem Squashing changes. So just created Diff from here and applied to new branch. 

I cannot make 2 commit for 2 jira tickets, since i already made code changes  and separating then would be a new code change for me and that wouldn't make sense to do.

**Comment by pushyamig** [comment]
Closing this as another is opened #1800 instead of this

**Pull request closed without merging**